packages/cli/src/environments/variables/variables.service.ee.ts

```diff
@@ -26,23 +23,25 @@ export class EEVariablesService extends VariablesService {
 		}
 	}
 
-	static async create(variable: Omit<Variables, 'id'>): Promise<Variables> {
+	async create(variable: Omit<Variables, 'id'>): Promise<Variables> {
 		if (!canCreateNewVariable(await this.getCount())) {
 			throw new VariablesLicenseError('Variables limit reached');
 		}
 		this.validateVariable(variable);
 
 		void Container.get(InternalHooks).onVariableCreated({ variable_type: variable.type });
-		return collections.Variables.save({
+		const saveResult = await collections.Variables.save({
 			...variable,
 			id: generateNanoId(),
 		});
+		await this.updateCache();
+		return saveResult;
 	}
 
-	static async update(id: string, variable: Omit<Variables, 'id'>): Promise<Variables> {
+	async update(id: string, variable: Omit<Variables, 'id'>): Promise<Variables> {
```

Comment:
`Omit<Variables, 'id'>` → To double check, is it ID-less when updating?

Reply:
presumably yes